docs/src/modules/components/AppFrame.js

```diff
@@ -113,9 +113,8 @@ const styles = (theme) => ({
     },
   },
   appBar: {
-    color: theme.palette.mode === 'light' ? null : '#fff',
-    backgroundColor: theme.palette.mode === 'light' ? null : theme.palette.background.level2,
     transition: theme.transitions.create('width'),
+    boxShadow: theme.shadows[4],
```

Comment:
This is suspicious. To me it rings as: something is not right in the app bar.

Reply:
Looking closer:

1. color: #272727 L 15% / #1D1D1D B 15%

<img width="339" alt="Capture d’écran 2021-06-02 à 14 14 03" src="https://user-images.githubusercontent.com/3165635/120478118-cca3a500-c3ac-11eb-841d-d3b71fa6f067.png">

https://deploy-preview-26545--material-ui.netlify.app/components/app-bar/#basic-app-bar

2. color: #202020 L 13% / #181818 B 12%

<img width="430" alt="Capture d’écran 2021-06-02 à 14 14 13" src="https://user-images.githubusercontent.com/3165635/120478168-db8a5780-c3ac-11eb-891e-3f40e0812ef7.png">

https://material.io/design/color/dark-theme.html

3. color: #2f2f2f L 18% / #232323 B 18%

<img width="423" alt="Capture d’écran 2021-06-02 à 14 14 57" src="https://user-images.githubusercontent.com/3165635/120478244-ee9d2780-c3ac-11eb-9360-7dae6e3ec43d.png">

https://deploy-preview-26545--material-ui.netlify.app/components/app-bar/#basic-app-bar

Reply:
It sounds like the background color for elevation={4} is a bit too light, but it's probably fine. Using elevation 8 sounds wrong.

Reply:
I have removed `elevation={8} and boxShadow`, use default elevation make more sense. I added it to make it looks the same as before.

Reply:
Sound great. In the future, I assume that we will need to remove the custom background color we use the the body of the documentation. It will be darker, creating more contrast.